src/models/loan/paymentCalculations.js

```diff
@@ -0,0 +1,46 @@
+function daysInYear(year) {
+  return 365
+}
+
+function daysInMonth(month) {
+  return 30.417
+}
+
+function effectiveMonthlyInterestRate(yearlyInterestRate, month, year) {
+  return ((1 + yearlyInterestRate / daysInYear(year)) ** daysInMonth(month)) - 1
+}
+
+const calculateRepaymentTerm = ({debt:{amountOwed, interestRate: {rate}}, paymentPlan: {monthlyPayment}}) => {
+  let repaymentTerm
```

Comment:
you don't really need to have a mutated variable here. you could just return from each of the conditionals